hri_multimodal_fusion/DetectionLabelArray   label_data
string                                      filepath
uint8                                       duration_sec
---